hack/aks/Makefile

```diff
@@ -116,13 +124,21 @@ endif
 	@$(MAKE) set-kubeconf
 
 overlay-byocni-nokubeproxy-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster without kube-proxy
+	$(AZCLI) network public-ip create --name $(IP_PREFIX)-$(CLUSTER) \
```

Comment:
I would recommend making `$(IP_PREFIX)-$(CLUSTER)` a variable, so you can use it in the cluster create command as well.